chromium_src/chrome/browser/ui/views/side_panel/search_companion/companion_page_browsertest.cc

```diff
@@ -0,0 +1,15 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "net/base/features.h"
+
+#include "base/test/scoped_feature_list.h"
+
+#define InitWithFeaturesAndParameters(A, B) \
+  InitWithFeaturesAndParameters(A, {net::features::kBraveHttpsByDefault})
```

Comment:
perhaps we can just disable the test, if it doesn't add any value for us now and in the future? Also there is not much sense in testing chromium features with our non-default configurations, b/c in reality we would never have the feature disabled, right?

Reply:
For now I'm leaving this test (and others) enabled because it's a good extra check that we haven't unintentionally broken anything from upstream. But I'm happy to re-visit this if it turns out to be a problem.